pkg/watch/watch.go

```diff
@@ -572,7 +615,9 @@ func processEvents(
 }
 
 func (o *WatchClient) CleanupDevResources(devfileObj parser.DevfileObj, out io.Writer) error {
-	fmt.Fprintln(out, "Cleaning resources, please wait")
+	spinner := log.Spinner("Cleaning resources, please wait")
```

Comment:
When `odo dev` exits, my terminal is in a bad state, the keys are not displayed on my command line (like in password mode), I need to run the `reset` command to make it work correctly again.

Reply:
Not sure if this is this spinner responsible.